app.js

```diff
@@ -17,10 +23,15 @@ class App {
   middlewares() {
     this.app.use(express.urlencoded({ extended: true }));
     this.app.use(express.json());
+    this.app.use(express.static(resolve(__dirname, 'uploads')));
   }
 
   routes() {
     this.app.use('/', homeRoutes);
+    this.app.use('/users/', userRoutes);
+    this.app.use('/tokens/', tokenRoutes);
+    this.app.use('/alunos/', alunoRoutes);
+    this.app.use('/photos/', photoRoutes);
```

Comment:
é interessante manter um padrão nas nomenclaturas, ou ingles ou portugues. No caso inglês é o padrão internacional, dessa forma ficaria algo como: 
```suggestion
    this.app.use('/users/', userRoutes);
    this.app.use('/tokens/', tokenRoutes);
    this.app.use('/students/', studentRoutes);
    this.app.use('/photos/', photoRoutes);
```